extensions/clusters/aks/aks_cluster_config.go

```diff
@@ -11,15 +11,19 @@ const (
 
 // ClusterConfig is the configuration needed to create an AKS host cluster
 type ClusterConfig struct {
+	AuthBaseURL                 *string           `json:"authBaseUrl,omitempty" yaml:"authBaseUrl,omitempty"`
 	AuthorizedIPRanges          *[]string         `json:"authorizedIpRanges,omitempty" yaml:"authorizedIpRanges,omitempty"`
 	AzureCredentialSecret       string            `json:"azureCredentialSecret" yaml:"azureCredentialSecret"`
+	BaseURL                     *string           `json:"baseUrl,omitempty" yaml:"baseUrl,omitempty"`
 	DNSPrefix                   *string           `json:"dnsPrefix,omitempty" yaml:"dnsPrefix,omitempty"`
+	HTTPApplicationRouting      *bool             `json:"httpApplicationRouting,omitempty" yaml:"httpApplicationRouting,omitempty"`
 	KubernetesVersion           *string           `json:"kubernetesVersion,omitempty" yaml:"kubernetesVersion,omitempty"`
 	LinuxAdminUsername          *string           `json:"linuxAdminUsername,omitempty" yaml:"linuxAdminUsername,omitempty"`
 	LinuxSSHPublicKey           *string           `json:"sshPublicKey,omitempty" yaml:"sshPublicKey,omitempty"`
 	LoadBalancerSKU             *string           `json:"loadBalancerSku,omitempty" yaml:"loadBalancerSku,omitempty"`
 	LogAnalyticsWorkspaceGroup  *string           `json:"logAnalyticsWorkspaceGroup,omitempty" yaml:"logAnalyticsWorkspaceGroup,omitempty"`
 	LogAnalyticsWorkspaceName   *string           `json:"logAnalyticsWorkspaceName,omitempty" yaml:"logAnalyticsWorkspaceName,omitempty"`
+	ManagedIdentity             **bool            `json:"managedIdentity,omitempty" yaml:"managedIdentity,omitempty"`
```

Comment:
Is this the expected type for this field?

Reply:
my only question as well.

Reply:
I had the same question :sweat: But that's what I noticed [here](https://github.com/rancher/rancher/blob/7c36ac3b730c4733434269bda75e2906955ef22c/pkg/client/generated/management/v3/zz_generated_aks_cluster_config_spec.go#L56), so I copied it, just like all the other fields that I added.

Reply:
@thardeck perhaps you can help with this. I see you are the author of https://github.com/rancher/rancher/pull/42388. Is `**bool` valid or is something amiss here? I noticed the code in the PR is autogenerated, but I am unsure about the source.

Reply:
It was more of a question, if this done in the generated code then this should be good to go. No need to involve Tim.

Reply:
> It was more of a question, if this done in the generated code then this should be good to go. No need to involve Tim.

@igomez06 @caliskanugur In that case can we merge the PR? It is kinda blocking my work.